app/rdf/query-cube-preview.ts

```diff
@@ -82,6 +82,7 @@ CONSTRUCT {
     ?cube cube:observationConstraint/sh:property ?dimension .
     ?dimension sh:path ?dimensionIri .
     OPTIONAL { ?dimension sh:datatype ?dimensionDataType . }
+    OPTIONAL { ?dimension sh:or/rdf:rest*/rdf:first/sh:datatype ?dimensionDataType . }
```

Comment:
What does it mean here ?

Reply:
Datatype could also be stored in such way, we used to retrieve it in a similar way [here](https://github.com/visualize-admin/visualization-tool/blob/2b77e5226adbdca7c43d592d57614a4791e696e7/app/rdf/parse.ts#L145-L149) – but this was cancelled, as we can avoid querying additional parameters and get data type directly from dimension values we fetch anyway 👍